src/server/capacity_unit_calculator.cpp

```diff
@@ -60,35 +60,56 @@ int64_t capacity_unit_calculator::add_write_cu(int64_t write_data_size)
     return write_cu;
 }
 
-void capacity_unit_calculator::add_get_cu(int32_t status, const dsn::blob &value)
+void capacity_unit_calculator::add_get_cu(int32_t status,
+                                          const dsn::blob &key,
+                                          const dsn::blob &value)
 {
-    if (status != rocksdb::Status::kOk && status != rocksdb::Status::kNotFound) {
+    if (status == rocksdb::Status::kNotFound) {
+        add_read_cu(1);
```

Comment:
Why do you add 1 to read_cu when key is not found?

Reply:
Calc cu need follow standards, so I just refer to aliyun.